app/src/renderer/vuex/modules/delegates.js

```diff
@@ -10,6 +10,7 @@ export default ({ node }) => {
     delegates: [],
     globalPower: null,
     loading: false,
+    loaded: false,
```

Comment:
why do we need `loaded` and `loading`?

Reply:
`loading`: the module has requested data,
`loaded`: the module has loaded data once so we can show "cached" data,

if we haven't `loaded` data yet and are currently `loading` we show an according message

Reply:
awesome.